cookbook/bundles/tests.md

```diff
@@ -0,0 +1,255 @@
+.. index::
+   single: Bundle; Testing third party bundles
+
+Writing tests for third party bundles
+=====================================
+
+Let's have a look at a minimized version of the Standard Edition's front controller:
```

Comment:
in order to make the next block a PHP code block, you have to use a double colon (`::`).

Also, "Let's" is a first person perspective, which is avoided in the docs.